plugins/node/instrumentation-dataloader/src/instrumentation.ts

```diff
@@ -242,10 +250,139 @@ export class DataloaderInstrumentation extends InstrumentationBase<DataloaderIns
         // .loadMany never rejects, as errors from internal .load
         // calls are caught by dataloader lib
         return original.call(this, ...args).then(value => {
+          span.setAttribute('cache.key', Array.from(args[0]));
+          span.setAttribute(
+            'cache.hit',
```

Comment:
Unfortunately these are not official OTEL semantic conventions, so I think we cannot add these here. I think we'll need `requestHook` and add these ourselves. For now we should remove this.

There are no cache conventions for OTEL yet. We're thinking about donating our conventions upstream for this.

Reply:
@AbhiPrasad - Should we add the `requestHook` with these updates, or in a separate PR?

Reply:
Let's add `requestHook` in a separate PR